docs/cypress/e2e/links.cy.ts

```diff
@@ -11,7 +11,7 @@ type EvtName =
   | 'RETURNING'
   | 'NO_HREF';
 let allLinks: string[] = [];
-const numberOfLinks = 119;
+const numberOfLinks = 121;
```

Comment:
@zhamujun Do we need to now update this manually every time a link is added? From [your PR](https://github.com/aws-amplify/amplify-ui/pull/2844): 
> Dynamic. When new links added or exist link changed, we don't have to update the tests.

Reply:
Yes, the links are dynamically tested, but the number of links has to be updated manually. There's no way for the test to know how many links there should be.

Reply:
The description in my PR was not accurate. Let me change that...

Reply:
Updated:
> Dynamic. When new links added or exist link changed, we don't have to update the tests. However, if a new page is added, we do need to update the number of pages being tested.